Support/Testing/CircleCI/Dockerfile

```diff
@@ -17,7 +17,9 @@
 #
 # This hash identifies the image you just built. You can then run it locally
 # with the following command:
-#     docker run --rm -it --name ds2-testing 360abf8e6246
+#     docker run --privileged --security-opt seccomp:unconfined --rm -it --name ds2-testing 360abf8e6246
```

Comment:
Do you need both `--privileged` and `--security-opt seccomp:unconfied`? I think I only needed the latter.

Also, are these the flags with which CircleCI runs our images?

Reply:
Yes, I believe these are the same flags.

I'm not sure if we need both flags. I was under the impression we need both, as they enable different capabilities.